config.go

```diff
@@ -50,9 +50,9 @@ type Config struct {
 // portion of this project. That is, the code responsible for querying Mesos,
 // et. al to gather metrics and send them to a "producer".
 type CollectorConfig struct {
-	HTTPProfiler bool                             `yaml:"http_profiler"`
-	Node         *node.NodeCollector              `yaml:"node,omitempty"`
-	MesosAgent   *mesos_agent.MesosAgentCollector `yaml:"mesos_agent,omitempty"`
```

Comment:
See underscore vs. package name comment above. For variables, camel case is fine. However, for files and package names the convention is lower case with underscores.